sycl/doc/extensions/SPIRV/SPV_INTEL_long_constant_composite.asciidoc

```diff
@@ -0,0 +1,235 @@
+SPV_INTEL_long_constant_composite
+=================================
+
+Name Strings
+------------
+
+SPV_INTEL_long_constant_composite
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Mariya Podchishchaeva, Intel
+- Alexey Sotkin, Intel
+- Ben Ashbaugh, Intel
+- Alexey Sachkov, Intel
+
+Notice
+------
+
+Copyright (c) 2020 Intel Corporation. All rights reserved.
+
+Status
+------
+
+Draft.
+
+Version
+-------
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2020-12-07
+| Revision           | 6
+|========================================
+
+Dependencies
+------------
+
+This extension is written against the SPIR-V Specification,
+Version 1.5 Revision 4, Unified
+
+This extension requires SPIR-V 1.0.
+
+Overview
+--------
+
+This extension adds new capability and instructions to allow to represent
+composite constants with number of Constituents greater than the maximum
+possible WordCount.
+
+Extension Name
+--------------
+
+To use this extension within a SPIR-V module, the following
+*OpExtension* must be present in the module:
+
+----
+OpExtension "SPV_INTEL_long_constant_composite"
+----
+
+Modifications to the SPIR-V Specification, Version 1.5
+------------------------------------------------------
+Capabilities
+~~~~~~~~~~~~
+
+Modify Section 3.31, "Capability", adding these rows to the Capability table:
+
+--
+[options="header"]
+|====
+2+^| Capability ^| Depends On
```

Comment:
In [the latest version of the spec](https://www.khronos.org/registry/spir-v/specs/unified1/SPIRV.html#_a_id_capability_a_capability) the last column titled as "Implicitly Declares"
```suggestion
2+^| Capability ^| Implicitly Declares
```